.github/workflows/notify-integration-release-via-manual.yaml

```
@@ -0,0 +1,50 @@
+# Manual release workflow is used for deploying documentation updates
+# on the specified branch without making an official plugin release. 
+name: Notify Integration Release (Manual)
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The release version (semver)"
+        default: 1.0.0
+        required: false
+      branch:
+        description: "A branch or SHA"
+        default: 'main'
+        required: false
+jobs:
+  notify-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      # Ensure that Docs are Compiled
+      - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+      - shell: bash
+        run: make generate
+      - shell: bash
+        run: |
+          if [[ -z "$(git status -s)" ]]; then
+            echo "OK"
+          else
+            echo "Docs have been updated, but the compiled docs have not been committed."
+            echo "Run 'make generate', and commit the result to resolve this error."
+            exit 1
+          fi
```

```suggestion
      - shell: bash
        run: make check-generate 
```

Similar here. `make check-generate` does more or less the same thing:

https://github.com/digitalocean/packer-plugin-digitalocean/blob/133cd6221b53e22de43c279e9b362336dc8d2546/GNUmakefile#L33-L42

---

For this one I would maybe suggest leaving the check here, but we can leverage `make check-generate` for it.
My rationale for keeping the check in is that this can be invoked on any branch in the repo, so we should check that the docs are up-to-date for this context, before sending the request to update to our doc servers.

---

Sounds good!